charts/guard/templates/auth-pads.yaml

```
@@ -0,0 +1,67 @@
+# PADS (PATH Auth Data Server) Helm Chart
+#
+# A gRPC server that provides Gateway Endpoint data from a configurable data source 
+# to the Go External Authorization Server, enabling authorization for the PATH Gateway.
+# 
+# Data sources can include:
+#   - YAML files
+#   - Postgres databases
+#
+# Github Repo: https://github.com/buildwithgrove/path-auth-data-server
+{{- if .Values.pads.enabled }}
+{{- $name := include "guard-helm.fullname" . }}
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{$name}}-pads
+  labels:
+    {{- include "guard-helm.labels" $ | nindent 4 }}
+spec:
+  replicas: {{ .Values.pads.replicas }}
+  selector:
+    matchLabels:
+      app: {{$name}}-pads
+  template:
+    metadata:
+      labels:
+        {{- include "guard-helm.labels" $ | nindent 8 }}
+        app: {{$name}}-pads
+    spec:
+      containers:
+      - name: path-auth-data-server
+        image: ghcr.io/buildwithgrove/path-auth-data-server:latest
+        imagePullPolicy: IfNotPresent
+        env: {{- toYaml .Values.pads.env | nindent 10 }}
+        ports:
+        - containerPort: {{ .Values.pads.port }}
+        volumeMounts:
+          - name: pads-data-volume
+            mountPath: /app/data
+      volumes:
+      - name: pads-data-volume
+        configMap:
+          name: pads-data
```

WDYT about making these values parametrized?

```suggestion
        volumeMounts:
          - name: {{$name}}-data
            mountPath: {{ .Values.pads.mountPath }}
      volumes:
      - name: {{$name}}-data
        configMap:
          name: pads-data
```

---

I'm for it; I will update in this PR.